test/suites/smoke/test-xcm-autopause.ts

```diff
@@ -1,21 +1,28 @@
 import "@moonbeam-network/api-augment/moonbase";
-import { describeSuite, expect } from "@moonwall/cli";
+import { ApiPromise } from "@polkadot/api";
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
 
 describeSuite({
   id: "S26",
   title: `XCM Mode should be equal to Normal`,
   foundationMethods: "read_only",
   testCases: ({ context, it, log }) => {
+    let paraApi: ApiPromise;
+
+    beforeAll(async function () {
+      paraApi = context.polkadotJs("para");
+    });
+
     it({
       id: "C100",
       title: "XCM Mode should be equal to Normal",
       test: async function (context) {
-        if ((await context.polkadotJs().consts.system.version.specVersion) < 3200) {
-          return context.skip();
+        if ((await paraApi.consts.system.version.specVersion).toNumber() < 3200) {
```

Comment:
```suggestion
        if ((paraApi.consts.system.version.specVersion).toNumber() < 3200) {
```

No need to await for consts